src/scenes/Root/Sidebar/SwooshIcon.tsx

```diff
@@ -0,0 +1,61 @@
+import { makeStyles, SvgIcon, SvgIconProps } from '@material-ui/core';
+import * as React from 'react';
+import { FC } from 'react';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '248px',
+    height: '136px',
+  },
+}));
+
+export const SwooshIcon: FC<SvgIconProps> = ({
+  classes: classesProp,
+  ...rest
+}) => {
+  const classes = useStyles();
+  return (
+    <SvgIcon
+      classes={{ root: classes.root, ...classesProp }}
```

Comment:
If `root` gets passed in it replaces our def which we don't want.
```suggestion
      classes={{ root: clsx(classes.root, classesProp.root), ...classesProp }}
```